collect_app/src/main/java/org/odk/collect/android/preferences/AggregatePreferencesActivity.java

```diff
@@ -55,6 +76,7 @@ protected void onPause() {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.HONEYCOMB)
```

Comment:
This is just a way to block Lint so I believe that the new code which uses features of an API level > 9 will result in a hard crash if the preferences screen is opened on devices that run Android versions below Honeycomb. Is that right?